app/api/v2/admin/entities/activity_with_user.rb

```diff
@@ -4,6 +4,9 @@ module API::V2::Admin
   module Entities
     class ActivityWithUser < API::V2::Entities::Base
       expose :user_ip, documentation: { type: 'String' }
+      expose :user_ip_country, documentation: { type: 'String' } do |activity|
```

Comment:
Let's add new field for the activity table, where we will define user_ip_country